go-tutorial.md

```diff
@@ -306,6 +305,29 @@ to the version you would like to use of Go.
 
 Next, we need a `BUILD.bazel` file in the root project directory.
 
+## Create the intial `deps.bzl` file
+
+In order for gazelle to run properly the first time, you may need to create a `deps.bzl` file in the root directory of
```

Comment:
It's not "to run properly" right? If you go with the default behavior then the deps go in WORKSPACE like you say later. So it's more about "if you want to keep your WORKSPACE shorter"